weather-forecast/src/components/InputField/InputField.tsx

```diff
@@ -1,18 +1,17 @@
 import React, { useEffect, useRef, useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { AppState } from '../../redux/rootReducer';
 import { getCurrentWeather } from '../../redux/weatherReducer';
+import { capitalize } from '../../utils/capitalize';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks';
```

Comment:
You can rename it to useDispatch and useSelector since only these hooks should be used to interact with redux store from components.